README.md

```diff
@@ -2,23 +2,26 @@
 
 MoveApps
 
-Github repository: *github.com/callumjclarke/Standardise_Formats_and_Calculate_Basic_Statistics*
+Github repository:
+<https://github.com/dmpstats/Standardise_Formats_and_Calculate_Basic_Statistics>
+
 
 ## Description
 
-This MoveApp contains settings to perform several basic cleaning processes and generation of extra columns of data. For example, using standardized names, binning the data to intervals of predefined duration and adding a speed column. This processing may also be useful for merging data from multiple studies of differing tag types. The user may specify any combination of the processes detailed below.
+This MoveApp contains settings to perform several basic cleaning processes and generation of extra columns of data. For example, using standardized names, thinning the data and adding a speed column. This processing may also be useful for merging data from multiple studies of differing tag types. The user may specify any combination of the processes detailed below.
 
 ## Documentation
 
 This MoveApp fulfills several purposes in a workflow. The overall aim is to standardize any input study into a format that can be merged at a later stage in a workflow with additional studies (that have also been processed) to avoid discrepancies in study names, units, data volume, and/or column names. In addition extra information such as location in UTMs, speed, distance travelled and time between points may be appended.
 
 The user may select any of the following settings:
 
--   Bin the data to intervals of X minutes
--   Create and append columns containing Universal Transverse Mercator (UTM) data
--   Create and append an *index* column, consisting of the animal's ID and timestamp concatenated
--   Create and append columns containing *speed*, *distance traveled* and/or *time since previous point* data
--   Create and append columns containing specific timestamp data (hour, minute and second, independently)
+-   Thinning the data by filtering one event per defined time window
+-   Create and append columns containing: 
+      - Universal Transverse Mercator (UTM) data
+      - An *index* column, consisting of the animal's ID and timestamp concatenated
+      - *speed*, *distance traveled* and/or *time since previous point* data
```

Comment:
is it time since previous point or time to next point?

Reply:
Yep, you're absolutely right! Will amend this and commit again